relay-server/src/services/spooler/sql.rs

```diff
@@ -155,3 +155,8 @@ pub async fn do_insert(
 
     Ok(count)
 }
+
+/// Creates DELETE statement which truncates the entire `envelopes` table.
+pub fn truncate<'a>() -> Query<'a, Sqlite, SqliteArguments<'a>> {
+    sqlx::query("DELETE FROM envelopes;")
```

Comment:
Maybe should add a quick comment that sqlite does have a truncate optimization on where (I had to look up why we're not using `TRUNCATE` which I know from other DBs)